chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1054,13 +1054,7 @@ impl PeerManagerActor {
                     NetworkResponses::RouteNotFound
                 }
             }
-            NetworkRequests::Challenge(challenge) => {
-                // TODO(illia): smarter routing?
-                self.state
-                    .tier2
-                    .broadcast_message(Arc::new(PeerMessage::Challenge(Box::new(challenge))));
-                NetworkResponses::NoResponse
-            }
+            NetworkRequests::Challenge(_) => NetworkResponses::NoResponse,
```

Comment:
Can we delete the `NetworkRequests::Challenge` variant instead of consuming it silently here?

Reply:
done.